climpred/bootstrap.py

```diff
@@ -51,6 +51,44 @@ def _resample(hind, resample_dim):
     return smp_hind
 
 
+def _resample_idx(init, bootstrap, dim='member', replace=True):
+    """Resample bootstrap times over dim."""
```

Comment:
Can you fill this out completely? It's a private function, but a really important one. E.g.

```python
"""Return resampled initialized ensemble with all iterations appended to a new dimension.

Args:
    init (xr object): Initialized ensemble to be resampled.
    bootstrap/iterations (int): Number of bootstrapping iterations to perform.
    dim (str): Dimension over which to resample. One of ['member', 'init']. Default 'member'.
    replace (bool): If True, resample with replacement. Default True.

Returns:
    Resampled dataset with iterations appended to new dimension ``bootstrap``.
"""
```